assets/src/pages/room/components/MediaControlButtons.tsx

```diff
@@ -93,7 +93,6 @@ const getAutomaticControls = (
         hideOnMobile: true,
         onClick: () => {
           local.screenShare.stop();
-          screenShare.setActive(false);
```

Comment:
Since we were stopping the screenShare track outright, it doesn't make sense to set any metadata for it.